docs/src/main/docs/microprofile/02_server-configuration.adoc

```diff
@@ -57,23 +58,29 @@ ssl:
     keystore-passphrase: "abcd"
 ----
 
+== Configure "admin" port
+For this example, we will use a `yaml` file. We configure a port 8011 that would server health and metrics endpoints,
+and then configure health and metrics to use it.
 
-For Helidon config, the default file is `application.yaml` on the classpath
- (e.g. src/main/resources/application.yaml).
+Alternative ports can be used by Server extensions that support it. This support is currently limited to MP Health and
+MP Metrics implementations in Helidon.
+
+You can create a custom Server extension and run routes on alternative ports - this support is limited to WebServer routing.
```

Comment:
Server extensions are not documented right ?
I think it's better omit this sentence for now.

Reply:
At whatever point this note is added, maybe add a sentence explaining the implications of the support being limited to WebServer routing...maybe as simple as "you cannot use JAX-RS" or something like that.

Reply:
I have removed the section about server extensions and clearly explained what it can be used for (I hope)